packages/strapi-helper-plugin/lib/src/utils/cleanData.js

```diff
@@ -0,0 +1,19 @@
+import { isArray } from 'lodash';
```

Comment:
```js
import { isArray, isObject } from 'lodash';

const cleanData = (value, key, secondKey) => {
  if (isArray(value)) {
    return value.map(obj => (obj[key] ? obj[key] : key));
  } else if (isObject(value)) {
    return value[key] || value[secondKey];
  } else {
    return value;  
  }
}

export default cleanData;
```

Reply:
Thanks for this one. A mistake though... 

```js
...
return value.map(obj => (obj[key] ? obj[key] : obj)); // We have to return the obj not the key :)
...
```

Reply:
Good catch, sorry for the mistake.